tests/Frameworks/Custom/OpenTracing/index.php

```diff
@@ -2,15 +2,11 @@
 
 require __DIR__ . '/vendor/autoload.php';
 
-try {
-    $otTracer = new \DDTrace\OpenTracer1\Tracer(\DDTrace\GlobalTracer::get());
-    $scope = $otTracer->startActiveSpan('web.request');
-    $span = $scope->getSpan();
-    $span->setTag('service.name', 'service_name');
-    $span->setTag('resource.name', 'resource_name');
-    $span->setTag('span.type', 'web');
-    $span->setTag('http.method', $_SERVER['REQUEST_METHOD']);
-    $span->finish();
-} finally {
-    \dd_trace_internal_fn("finalize_telemetry");
```

Comment:
Calling `finalize_telemetry` manually causes the metrics (like `spans_created`) to be sent twice.
Once with `service_name` as service name, the other with `index.php`

Reply:
I think back in the old times finalize_telemetry was _final_ :-) (But I think that matters no longer.)